### PR TITLE
docs(aio): Fix typos, formatting, add screenshot for Universal guide

### PR DESCRIPTION
**Pull request by fmalcher**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features) **not applicable**
- [x] Docs have been added / updated (for bug fixes / features) 

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the new behavior?

This is about the Universal guide in AIO.
- Fix some typos
- Improve some formatting, but very small things
- add screenshot for throttling in Chrome

I built it locally, looked good. 😎

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/guide/universal.md [hunk 1]
```diff
@@ -571,10 +568,12 @@ better appreciate the launch-speed advantage of a universal app running on a low
 
 Open the Chrome Dev Tools and go to the Network tab.
 Find the [Network Throttling](https://developers.google.com/web/tools/chrome-devtools/network-performance/reference#throttling) dropdown on the far right of the menu bar.
-
 Try one of the "3G" speeds.
 The server-rendered app still launches quickly but the full client app may take seconds to load.
 
+<img src="generated/images/guide/universal/chrome-throttling.png" alt="Network Throttling in Chrome" width="1000">
```
Comment (gkalpak) [inline aio/content/guide/universal.md]: Why `width="1000"`? I think this won't render well for specific screen sizes. (It should be around 600 iirc).

Comment (fmalcher) [reply aio/content/guide/universal.md]: Thanks, good point. I changed this to `class="left"` like all the other images have.
Looks good for every screen size now (tried it with the Chrome Devtools Device Emulator)

**Comment by brandonroberts** [comment]
@fmalcher should this be closed in favor of #24375 ?

**Comment by fmalcher** [comment]
@brandonroberts Yep, sounds legit. I accidentally included the commit from this PR into #24375, so they're redundant. Closing this!

**Pull request closed without merging**